doc/multiprecision.qbk

```diff
@@ -1091,57 +1108,85 @@ the default by defining either `BOOST_MP_USE_FLOAT128` or `BOOST_MP_USE_QUAD`.
 * When the underlying type is Intel's `_Quad` type, the code must be compiled with the compiler option `-Qoption,cpp,--extended_float_type`.
 * When compiling with `gcc`, you need to use the flag `--std=gnu++11/14/17`, as the suffix 'Q' is a GNU extension. Compilation fails with the flag `--std=c++11/14/17`
 unless you also use `-fext-numeric-literals`.
+* You will need to link to `libquadmath.dll` with the link command `-lquadmath` and ensure that the DLL is visible by the linker. 
+If you are using the B2/bjam build system then commands`<linkflags>-lQUADMATH` and `<linkflags>-L"path/to/lib"` will be needed.
+* The values shown by `std::numeric_limits<float128>` and extremely close ['but not identical]
+to those from the equivalent precision and range multiprecision types `std::numeric_limits<cpp_bin_float_quad>` and `std::numeric_limits<cpp_dec_float_quad>`.
```

Comment:
Actually, I hope they should be exactly the same, I haven't checked though have you?

Reply:
When I previously checked a few, some appeared a bit different in the 'noisy' digits.  I have rechecked and now understand that this was an aberration caused by a previous difference in max_digits10 (now both max_digits10=36) and one display showing trailing zeros, and the other not.  (support for std::hex would help with this).

Now looks exact, but I haven't done an exhaustive check.   (should I) So now changed to:

* The values shown by `std::numeric_limits<float128>` are the same as those from the equivalent precision and range multiprecision types `std::numeric_limits<cpp_bin_float_quad>` and `std::numeric_limits<cpp_dec_float_quad>`.

I also propose to add an example program and include a listing all the numeric limits values for a variety of types, including these of course.  I feel users will find these convenient.